packages/webpack-cli/lib/groups/runVersion.js

```diff
@@ -53,7 +53,7 @@ const outputVersion = (args) => {
             const { name, version } = require(`${usedCommands[0].packageName}/package.json`);
             logger.raw(`${name} ${version}`);
         } catch (e) {
-            logger.error('Error: External package not found.');
+            logger.error(`Error: package '${usedCommands[0].packageName}' not found.`);
```

Comment:
codecove says what we don't have tests on this, let's add this

Reply:
Any suggestions on how to test it?

Reply:
mocking